src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -55,7 +55,7 @@ public void InaccessibleFieldRequestedByPolicy(string dbPolicy)
             RuntimeConfigValidator configValidator = new(provider, fileSystem, new Mock<ILogger<RuntimeConfigValidator>>().Object);
 
             // Assert that expected exception is thrown.
-            DataApiBuilderException ex = Assert.ThrowsException<DataApiBuilderException>(() => configValidator.ValidatePermissionsInConfig(runtimeConfig));
+            DataApiBuilderException ex = Assert.ThrowsException<DataApiBuilderException>(() => RuntimeConfigValidator.ValidatePermissionsInConfig(runtimeConfig));
```

Comment:
Now that `ValidatePermissionsInConfig` is static, the block of code above this seems unneeded now: 
```csharp
            MockFileSystem fileSystem = new();
            RuntimeConfigLoader loader = new(fileSystem);
            RuntimeConfigProvider provider = new(loader);
            RuntimeConfigValidator configValidator = new(provider, fileSystem, new Mock<ILogger<RuntimeConfigValidator>>().Object);
```

Reply:
thanks!. fixed.